mllib/src/main/scala/org/apache/spark/mllib/stat/impl/MultivariateGaussian.scala

```diff
@@ -30,33 +31,67 @@ import org.apache.spark.mllib.util.MLUtils
  * @param mu The mean vector of the distribution
  * @param sigma The covariance matrix of the distribution
  */
-private[mllib] class MultivariateGaussian(
+class MultivariateGaussian private[mllib] (
```

Comment:
If we want to make it public, should it still under `stat.impl`? Maybe is it good to put it under `stat.distribution` and mark it as `@DeveloperApi` since we don't have a `Distribution` trait yet.